src/renderer/navigation/navigate.injectable.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import type { LocationDescriptor } from "history";
+import { action } from "mobx";
+import observableHistoryInjectable from "./observable-history.injectable";
+import { createPath } from "history";
+
+export type Navigate = (location: LocationDescriptor) => void;
+
+const navigateInjectable = getInjectable({
```

Comment:
Is this an alternate way of navigation? That's not ok, use `navigateToRouteInjectable` or rarely `navigateToUrlInjectable`.

Reply:
This is the same as `nativateToUrlInjectable` I guess since it is for the extension API and contexts.

Reply:
So it is not quite the same as `navigateToUrl` because it doesn't support the options

Reply:
The options? What are they used for? Can eg. `nativateToUrlInjectable` adapt to this?

Reply:
Ok. Resolved.